README.md

```diff
@@ -1,98 +1,144 @@
 # vcenter-connector
 
-vcenter-connector is an OpenFaaS event connector built to consume events from vCenter from VMware
+vcenter-connector is an OpenFaaS event connector built to consume events from vCenter from VMware vSphere.
 
-With this project your functions can subscribe to events generated by the changes in your vCenter installation - for instance a VM being created, turned on or deleted. This allows you to extend vCenter's functionality by writing functions to execute each time an event is fired. An example may be tagging a VM with the date it was last turned on or applying a tag showing which user made a change to an object. 
+With this project your functions can subscribe to events generated by the changes (i.e. events) in your vCenter installation - for instance a VM being created, turned on or deleted. This allows you to extend vCenter's functionality by writing functions to execute each time an event is fired. An example may be tagging a VM with the date it was last turned on or applying a tag showing which user made a change to an object. 
 
 ## Status
 
 This project uses the [OpenFaaS Connector SDK](https://github.com/openfaas-incubator/connector-sdk).
 
-The code is under active development and only suitable for early adopters. More examples and instructions for use in Kubernetes are being worked on.
+The code is under **active development** and only suitable for early adopters. More examples and instructions for use in Kubernetes are being worked on, including improved handling of secrets (accounts and passwords).
```

Comment:
```
including improved handling of secrets (accounts and passwords)
```

Please can you rather say the following (which doesn't generalise)

```
For the initial version the vCenter admin credentials need to be stored in plain-text in your YAML files, but this will move to using Kubernetes secrets in the next version.
```